index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header-box">
+        <p>John Doe</p>
```

Comment:
- [ ] Nice job adding a logo to your page 👍. Please consider wrapping the logo in an `a` tag instead of a `p` as the logo should be a link that leads to the homepage. See my suggestion below.

```suggestion
      <header class="header-box">
        <a href="#">John Doe</a>
```